docs/gettsim_objects/functions.rst

```diff
@@ -9,6 +11,7 @@ further below. By clicking on the ``source`` hyperlink next to the function sign
 you can see the full source code of the function.
 
 .. automodsumm:: gettsim.functions
```

Comment:
In the code documentation part, gettsim is in lower case, in contrast to GETTSIM in the rest of the documentation. That should be harmonized